common/persistence/execution_manager.go

```diff
@@ -160,72 +160,6 @@ func (m *executionManagerImpl) UpdateWorkflowExecution(
 	return &UpdateWorkflowExecutionResponse{MutableStateUpdateSessionStats: msuss}, err1
 }
 
-func (m *executionManagerImpl) SerializeExecutionInfo(
```

Comment:
There is also an deserialize execution info?

Reply:
does not seem to have that.